### ISSUE TITLE
[F.Rq.] N64 VRU (microphone) peripheral support

### ISSUE THREAD

**Issue by getCursorsExe** [TASEmulators/BizHawk#2746]
BizHawk literally needs microphone support. There are several games that are IMPOSSIBLE to complete normally without it. For example, "Hey You, Pikachu!" on Nintendo 64 is impossible to play due to lack of microphone support, and we need to modify some hex values to continue. Another example is with DS games (I know that this core is in Beta), where most games require Mic to play, but we can't because of lack of microphone support.

**Comment by YoshiRulz** [comment]
melonDS is already tracked as #1914.

Mupen upstream doesn't support *Hey You, Pikachu!*: mupen64plus/mupen64plus-core#157, closing as "Waiting on Upstream".

**Comment by Ecksters** [comment]
Mupen has implemented support in core for the VRU, Simple64 is the only GUI implementation that has support for it to my knowledge though.

**Comment by DerekTurtleRoe** [comment]
Rosalie's Mupen GUI is also adding support for it as well:

https://github.com/Rosalie241/RMG/releases/tag/v0.4.2
